src/components/CautionaryContact/CautionaryContactTable.test.js

```diff
@@ -1,16 +1,25 @@
 import { render } from '@testing-library/react'
 import UserContext from '../UserContext'
-import CautionaryContactView from './CautionaryContactView'
+import CautionaryContactTable from './CautionaryContactTable'
 import { operative } from 'factories/operative'
-describe('CautionaryContactView component', () => {
-  const query = {
-    cautContact: ['VA', 'DAT', 'CIT'],
+import { CAUTIONARY_CONTACT } from '../../utils/cautContactAlertsDescription'
+
+describe('CautionaryContactTable component', () => {
+  const props = {
+    cautionaryContacts: CAUTIONARY_CONTACT,
+    query: {
+      cautContact: ['VA', 'DAT', 'CIT'],
+    },
   }
+
   describe('when logged in as an operative and has NO queries', () => {
     it('should render properly: with NO highlithed codes', () => {
       const { asFragment } = render(
         <UserContext.Provider value={{ user: operative }}>
-          <CautionaryContactView />
+          <CautionaryContactTable
+            cautionaryContacts={props.cautionaryContacts}
+            query={false}
```

Comment:
the `query={false}` is not required, as we're checking on `CautionaryContactTable` if we `query` exists